### PR TITLE
Mass Refactoring of Messages

### PR DESCRIPTION
**Pull request by muhammad-khair**
### Related Issues:
- #261 
- #284

### CODE REVIEW

**Comment by muhammad-khair** [comment]
Converted back to draft to wait for review with team

File: src/main/java/seedu/tuitione/logic/commands/EnrollCommand.java [hunk 1]
```diff
@@ -68,21 +64,26 @@ public CommandResult execute(Model model) throws CommandException {
...
+            alertMessageContainer = String.format(STUDENT_ALREADY_ENROLLED_CONSTRAINT,
+                    studentToEnroll.getName(), lesson);
 
-        // final overall check
-        if (!lesson.isAbleToEnroll(studentToEnroll)) {
-            throw new CommandException(String.format(MESSAGE_UNABLE_TO_ENROLL, studentToEnroll.getName(), lesson));
+        } else if (lesson.doesStudentHaveConflictingTimings(studentToEnroll)) {
+            alertMessageContainer = String.format(CONFLICTING_TIMINGS_CONSTRAINT, studentToEnroll.getName(), lesson);
+
+        } else if (!lesson.isStudentOfSameGrade(studentToEnroll)) {
```
Comment (eezj35) [inline src/main/java/seedu/tuitione/logic/commands/EnrollCommand.java:79]: I think grade checking should be done before time checking

Comment (muhammad-khair) [reply src/main/java/seedu/tuitione/logic/commands/EnrollCommand.java:76]: Good point! Will update it

**Review by eezj35 — APPROVED** [review]
LGTM!

**Pull request merged**